processes/host-selection/host-prioritization.md

```diff
@@ -0,0 +1,37 @@
+# Meeting to decide host prioritization
+
+**DATE:** 2019-23-01
+
+**PARTICIPANTS:** @Jethet, @eskape16, @daverick, @amaliacardenas
+**LOCATION:** Cloud Co-working
+
+## TL;DR
+We want to be more transparent with the way we organize events. In this way anyone who attends or coaches at codebar knows our needs and if interested can help us out. The reason why we want to do this is to have more members from the community participate in the organization of events and to help us as we expand.
+
+In this document, you will find
+1. The criteria that we use when we prioritize where to host codebar events.
+2. A guide for how to contact hosts who want to sponsor a workshop.
+
+# Host prioritization criteria
+
+## Must Have
+* **Aligns with the codebar mission**
+* **Safe place**
+* **Food and drinks**
+* **Comfortable space for workshops**
+
+## Important
+* **Accessibility within the city**
+* **Maximum capacity**
+
+## Nice to have
+* **Comfortable space for talks**
+* **Sustainable company**
+* **Music at the start of workshops**
```

Comment:
This is a lovely idea. At the same time, I see potential difficulties. To be very practical: we will need someone in charge of the music. In my experience, things like fading out music, getting microphones in order etc. usually go wrong and you get a messy start of your workshop. Also, who decides what kind of music? Beethoven or Ladysmith Black Mambazo? Music conveys a message and we would have to select music that does not offend anyone.
Suggestion: unless we feel that we have a problem with awkward silent moments, it is probably easier to let people create their own noise.